src/events/SDL_pen.c

```diff
@@ -363,10 +365,45 @@ void SDL_SendPenTouch(Uint64 timestamp, SDL_PenID instance_id, const SDL_Window
             event.ptouch.down = down;
             SDL_PushEvent(&event);
         }
+
+        SDL_Mouse *mouse = SDL_GetMouse();
+        if (mouse && mouse->pen_mouse_events) {
+            if (window) {
```

Comment:
Should you move this into the previous conditional like you did below?

Reply:
Cleaned this up better in 82d97706d6e78d10d17dd81441d5af9cf09aff25.